code/game/objects/items/weapons/melee/melee_misc.dm

```diff
@@ -45,6 +45,388 @@
+	lefthand_file = 'icons/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons_righthand.dmi'
+	icon_state = "snakesfang"
+	item_state = "snakesfang"
+	flags = CONDUCT
+	force = 25
+	throwforce = 10
+	w_class = WEIGHT_CLASS_BULKY
+	sharp = TRUE
+	origin_tech = "combat=6;syndicate=3"
+	attack_verb = list("slashed", "sliced", "chopped")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	materials = list(MAT_METAL = 1000)
+	needs_permit = TRUE
+
+/obj/item/melee/snakesfang/examine_more(mob/user)
+	. = ..()
+	. += "A uniquely curved, black and red sword. Extra-edgy and cutting-edge."
+	. += ""
+	. += "The MK-IV Enhanced Combat Blade, more colloquially known as the ‘Snakesfang’, is a vicious yet stylish weapon designed \
+	by a relatively unknown weapons forge with known ties to the Syndicate. With a wide, curved blade and dual points \
+	resembling the fangs of the organization’s serpent motif, the Snakesfang is a statement like no other."
+	. += ""
+	. += "While the benefits of its unique design are dubious at best, the Snakesfang is undoubtedly a perilous weapon, with a hardened \
+	plastitanium edge that can cause untold harm to a soft target. In the right hands, it can be a terrifying weapon to behold, \
+	and it’s said that blood runs down the blade in just the right way, to drip artfully from the twin ‘fangs’ at its apex."
+
+/obj/item/melee/snakesfang/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/parry, _stamina_constant = 2, _stamina_coefficient = 0.5, _parryable_attack_types = NON_PROJECTILE_ATTACKS)
+
+// Unathi Sword
+/obj/item/melee/breach_cleaver
+	name = "breach cleaver"
+	desc = "Massive, heavy, and utterly impractical. This sharpened chunk of steel is too big and too heavy to be called a sword."
+	lefthand_file = 'icons/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons_righthand.dmi'
+	base_icon_state = "breach_cleaver"
+	icon_state = "breach_cleaver0"
+	flags = CONDUCT
+	force = 5
+	throwforce = 5
+	w_class = WEIGHT_CLASS_BULKY
+	sharp = TRUE
+	origin_tech = "combat=6;syndicate=5"
+	attack_verb = list("slashed", "cleaved", "chopped")
+	hitsound = 'sound/weapons/swordhitheavy.ogg'
+	materials = list(MAT_METAL = 2000)
+	needs_permit = TRUE
+	///How much damage the sword does when wielded
+	var/force_wield = 40
+
+/obj/item/melee/breach_cleaver/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/two_handed, force_wielded = force_wield, force_unwielded = force, icon_wielded = "[base_icon_state]1", wield_callback = CALLBACK(src, PROC_REF(wield)), unwield_callback = CALLBACK(src, PROC_REF(unwield)))
+	AddComponent(/datum/component/parry, _stamina_constant = 2, _stamina_coefficient = 0.5, _parryable_attack_types = NON_PROJECTILE_ATTACKS)
+
+/obj/item/melee/breach_cleaver/examine(mob/user)
+	. = ..()
+	if(isAntag(user))
+		. += "<span class='warning'>When wielded, this blade has different effects depending on your intent, similar to a martial art. \
```

Comment:
does this really need to be a `warning` class? Why not just notice or no class at all?